quinn-proto/src/lib.rs

```diff
@@ -17,6 +17,7 @@
 // Fixes welcome:
 #![allow(clippy::cognitive_complexity)]
 #![allow(clippy::too_many_arguments)]
+#![allow(clippy::len_without_is_empty)]
```

Comment:
many lint changes in these later PRs.

clippy warns on this because of we make the packet length method public for fuzz test. I'm unsure to ignore outright. Probably better to either implement the `is_empty()` or make the `.len()` private again and remove from packet fuzz target. Any preference?

Reply:
I think ignoring the lint is right, but let's scope it tightly to the particular places it complains, rather than globally for the entire library?